config/renovate/renovate.json

```diff
@@ -78,10 +78,28 @@
       }
     ]
   },
+  "customManagers": [
+    {
+      "customType": "regex",
+      "description": "Fedora updates from a stable source",
+      "fileMatch": ["(^|/|\\.)([Dd]ocker|[Cc]ontainer)file$", "(^|/)([Dd]ocker|[Cc]ontainer)file[^/]*$"],
+      "matchStrings": ["(?<depName>[\\w\\.\\-\\/]+\\/fedora[\\w\\-]*):(?<currentValue>[0-9]+)"],
+      "packageNameTemplate": "fedora",
+      "datasourceTemplate": "endoflife-date"
+
+    }
+  ],
   "dockerfile": {
     "enabled": true,
     "additionalBranchPrefix": "{{baseBranch}}/",
-    "branchPrefix": "konflux/mintmaker/"
+    "branchPrefix": "konflux/mintmaker/",
+    "packageRules": [
+      {
+        "description": "Don't update Fedora to an unstable version",
+        "matchPackagePatterns": ["[\\w\\.\\-\\/]+\\/fedora[\\w\\-]*"],
+        "enabled": false
```

Comment:
I just realized if users are using tag + digest pinning, it will break after we disabling this, the new regex manager doesn't support it, right?